node/DEVELOPER.md

```diff
@@ -107,16 +107,38 @@ Before starting this step, make sure you've installed all software requirments.
 5. Integrating the built GLIDE package into your project:
    Add the package to your project using the folder path with the command `npm install <path to GLIDE>/node`.
 
--   For a fast build, execute `npm run build`. This will perform a full, unoptimized build, which is suitable for developing tests. Keep in mind that performance is significantly affected in an unoptimized build, so it's required to build with the `build:release` or `build:benchmark` option when measuring performance.
--   If your modifications are limited to the TypeScript code, run `npm run build-external` to build the external package without rebuilding the internal package.
--   If your modifications are limited to the Rust code, execute `npm run build-internal` to build the internal package and generate TypeScript code.
--   To generate Node's protobuf files, execute `npm run build-protobuf`. Keep in mind that protobuf files are generated as part of full builds (e.g., `build`, `build:release`, etc.).
+6. Testing the GLIDE npm package locally:
+   The `node/npm/glide` folder contains a package wrapper that re-exports all the native bindings. To build and test this package, follow these steps:
+
+    ```bash
+
+     Build node package:
+     cd node
+     export nativeStr=darwin-x64; export scope=@valkey/;
+     envsubst < package.json.tmpl > "package.json"
+     npm run build
+     npm run build:release
+
+     In npm/glide package run the following commands:
+     cd ../npm/glide
+     export pkg_name=valkey-glide; export package_version=99.99.0; export scope=@valkey/;
+     envsubst < package.json.tmpl > "package.json"
+     npm run build
+     npm run build:test
+     npm run test
+     npm run test -- --testNamePattern="Exported symbols test"
```

Comment:
Isn't this test run in scope of `npm run test` task?

Reply:
Yes, I will add a comment, for running a specific test.